src/monitor/dto/update-monitor.dto.ts

```diff
@@ -0,0 +1,50 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsOptional, IsString, MaxLength } from 'class-validator';
+
+export type hours = {
```

Comment:
Tipo nunca se cria com letra minúscula:

```suggestion
export type AvailabilityDayPeriod = {
```